integration_tests/commands/http/json_test.go

```diff
@@ -647,6 +647,38 @@ func TestJsonStrlen(t *testing.T) {
 			},
 			expected: []interface{}{"OK", []interface{}{float64(5), float64(3)}},
 		},
+		{
+			name: "STRLEN with no path and object at root",
+			commands: []HTTPCommand{
+				{Command: "JSON.SET", Body: map[string]interface{}{"key": "k", "path": "$", "json": map[string]interface{}{"name": "bhima", "age": 10}}},
+				{Command: "JSON.STRLEN", Body: map[string]interface{}{"key": "k"}},
+			},
+			expected: []interface{}{"OK", "WRONGTYPE wrong type of path value - expected string but found object"},
+		},
+		{
+			name: "STRLEN with no path and object at boolean",
+			commands: []HTTPCommand{
+				{Command: "JSON.SET", Body: map[string]interface{}{"key": "k", "path": "$", "json": true}},
+				{Command: "JSON.STRLEN", Body: map[string]interface{}{"key": "k"}},
+			},
+			expected: []interface{}{"OK", "WRONGTYPE wrong type of path value - expected string but found boolean"},
+		},
+		{
+			name: "STRLEN with no path and object at array",
+			commands: []HTTPCommand{
+				{Command: "JSON.SET", Body: map[string]interface{}{"key": "k", "path": "$", "json": []int{1, 2, 3, 4}}},
+				{Command: "JSON.STRLEN", Body: map[string]interface{}{"key": "k"}},
+			},
+			expected: []interface{}{"OK", "WRONGTYPE wrong type of path value - expected string but found array"},
+		},
+		{
+			name: "STRLEN with no path and object at number",
+			commands: []HTTPCommand{
+				{Command: "JSON.SET", Body: map[string]interface{}{"key": "k", "path": "$", "json": 1}},
+				{Command: "JSON.STRLEN", Body: map[string]interface{}{"key": "k"}},
+			},
+			expected: []interface{}{"OK", "WRONGTYPE wrong type of path value - expected string but found number"},
```

Comment:
```suggestion
			expected: []interface{}{"OK", "WRONGTYPE wrong type of path value - expected string but found integer"},
```

Reply:
can you elaborate more on this? Because in JSON all integers are considered as floats so we are getting it as numberType.

Reply:
Yes, so we can create a wrapper function around the current function you are using to ensure both numbers and integer resolve to integer. Another way would be to use simple if - else to achieve this.